langchain4j-core/src/main/java/dev/langchain4j/store/embedding/EmbeddingStore.java

```diff
@@ -113,4 +113,40 @@ default List<EmbeddingMatch<Embedded>> findRelevant(
         throw new RuntimeException("Not implemented");
     }
 
+    /**
+     * Finds the most relevant (closest in space) embeddings to the provided reference embedding by specified partitions.
+     * Currently only milvus supports it.
+     *
+     * <p>Default implementation throws an exception.
+     *
+     * @param referenceEmbedding The embedding used as a reference. Returned embeddings should be relevant (closest) to this one.
+     * @param maxResults         The maximum number of embeddings to be returned.
+     * @param partitionNames     The range of partitions used for querying, Can be empty.
+     * @return A list of embedding matches.
+     * Each embedding match includes a relevance score (derivative of cosine distance),
+     * ranging from 0 (not relevant) to 1 (highly relevant).
+     */
+    default List<EmbeddingMatch<Embedded>> findRelevant(Embedding referenceEmbedding, int maxResults, List<String> partitionNames) {
```

Comment:
I would suggest placing this method on the `MilvusEmbeddingStore` as this is not necessarily applicable to all other implementations.

Reply:
I agree with your suggestion. 
My local version is under the milvus package: extending EmbeddingStore to MilvusEmbeddingStore, and then implementing MilvusEmbeddingStore.
Is this solution feasible? I'm doing local testing and will submit a PR after passing it.

Reply:
@igerdy yes, this makes sense!

Reply:
okay, I'll submit this part of the code first.